demo-simple/src/main/java/com/novoda/downloadmanager/demo/MainActivity.java

```diff
@@ -40,7 +43,8 @@
     private static final DownloadFileId FILE_ID_1 = DownloadFileIdCreator.createFrom("file_id_1");
     private static final String FIVE_MB_FILE_URL = "http://ipv4.download.thinkbroadband.com/5MB.zip";
     private static final String TEN_MB_FILE_URL = "http://ipv4.download.thinkbroadband.com/10MB.zip";
-    private static final String TWENTY_FILE_URL = "http://ipv4.download.thinkbroadband.com/20MB.zip";
+    private static final String TWENTY_MB_FILE_URL = "http://ipv4.download.thinkbroadband.com/20MB.zip";
+    private static final String FIFTHY_MB_FILE_URL = "http://ipv4.download.thinkbroadband.com/50MB.zip";
```

Comment:
typo `FIFTY` also not used?

Reply:
https://github.com/novoda/download-manager/pull/318/commits/ba90d9f436d912b9b56b09b81c2b52cecaf09052